### PR TITLE
Removing Windows from OpenCL tests - avoid fails on develop

### PR DESCRIPTION
**Pull request by rok-cesnovar**
## Summary

There are some issue with the Windows OpenCL tests on Jenkins. Develop tests are failing due to that. I am unable to reproduce it locally on Windows so I am going to remove the Windows machine from tests for now and then investigate.

## Tests

/

## Side Effects

/

## Checklist

- [x] Copyright holder: Rok Češnovar

    The copyright holder is typically you or your assignee, such as a university or company. By submitting this pull request, the copyright holder is agreeing to the license the submitted work under the following licenses:
      - Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
      - Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

- [x] the basic tests are passing

    - unit tests pass (to run, use: `./runTests.py test/unit`)
    - header checks pass, (`make test-headers`)
    - dependencies checks pass, (`make test-math-dependencies`)
    - docs build, (`make doxygen`)
    - code passes the built in [C++ standards](https://github.com/stan-dev/stan/wiki/Code-Quality) checks (`make cpplint`)

- [x] the code is written in idiomatic C++ and changes are documented in the doxygen

- [x] the new changes are tested

### CODE REVIEW

**Comment by rok-cesnovar** [comment]
@serban-nicusor-toptal Can we boot the AWS GPU instance back up the way we had it before? Given that the gelman-group-linux is offline.

**Comment by serban-nicusor-toptal** [comment]
Hey, sure! It's gonna be up in just a few moments.

File: Jenkinsfile [hunk 1]
```diff
@@ -268,7 +268,7 @@ pipeline {
                     }
                 }
                 stage('OpenCL GPU tests') {
-                    agent { label "gelman-group-win2 || linux-gpu" }
+                    agent { label "inux-gpu" }
```
Comment (serban-nicusor-toptal) [inline Jenkinsfile:271]: This might be an unintended typo.

Comment (rok-cesnovar) [reply Jenkinsfile]: Yeah, silly mistake :) Thanks. Should be good to approve now.

**Review by serban-nicusor-toptal — APPROVED** [review]
LGTM

**Pull request merged**